BlogPost.md

```diff
@@ -0,0 +1,160 @@
+# Deploying your application to StarlingX
+
+Deploying your own application to StarlingX is as easy as any other Kubernetes
+deployment out there. In this blog post we'll show you the process with a
+simple demonstration application.
+
+It's important to understand an application can be deployed in many ways to
+the Kubernetes cluster(s) that StarlingX manages:
+
+- [raw Kubernetes](https://kubernetes.io/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/);
+- [Helm](https://helm.sh/docs/intro/using_helm/#helm-install-installing-a-package);
+- [Flux](https://fluxcd.io/); and finally
+- StarlingX Application, which benefits from tight integration with the
+  [StarlingX system](https://opendev.org/starlingx/config).
+
+> _NOTE_: TODO Saying that Helm is the most popular package manager needs a reference/link.
+
+In this particular demonstration we will focus on [Helm](https://helm.sh/),
+which is the most popular package manager for Kubernetes. Future blog posts
+will address other deployment types and their advantages.
+
+Any developer that already packages their application with
+[Helm](https://helm.sh/) will be able to deploy their application in
+StarlingX without any hassle.
+
+## The Demo App
+
+The application simulates a network of antivirus scanners and threat-monitoring
+running independently and geographically distributed called `nodes`. When a node
+detects threats above a configurable threshold in an area it reports to a
+`central` instance responsible for aggregating, processing, and storing data
+from all the `nodes` while presenting a bird's-eye view in a simple dashboard.
+
+The application can act either as a `node` or a `central` (which are called
+"personalities") depending on the environment variables it has available.
+
+- `Nodes` are geographically spread out doing the actual work of monitoring, for
+  example, local networks.
+- `Central` is a deployment that receives messages from `nodes`, processing and
+  presenting data related to them.
+
+It's also possible to configure the threshold of a `node`, which determines
+their sensitivity in the scanner for threats.
+
+## Deploying the Demo App
+
+### Via a Helm Package file
+
+> _TODO_:  this is a WIP section. Similar to https://docs.starlingx.io/deploy_install_guides/release/kubernetes_access.html#id3
+
+You can simply use the packaged Helm chart from your own source code, and
+use the [Helm CLI](needs-reference) to install the package.
+
+#### Package the Helm chart
+
+```shell
+git checkout https://github.com/bmuniz-daitan/poc-starlingx-messages.git
+cd poc-starlingx-messages
+helm package helm-chart/
+```
+
+The above commands should create a `poc-starlingx-<version>.tgz` file.
+
+```shell
+...
+Successfully packaged chart and saved it to: .../PoC-StarlingX/poc-starlingx-1.5.2.tgz
```

Comment:
shouldn't it be .../poc-starlingx-messages/poc-starlingx-1.5.2.tgz?

Reply:
It should. Copied from my setup where the folder is `PoC-StarlingX`. :facepalm: